assets/js/googlesitekit/notifications/components/layout/SubtleNotification.js

```diff
@@ -43,6 +43,7 @@ const SubtleNotification = forwardRef(
 			description,
 			dismissCTA,
 			additionalCTA,
+			reverseCTAs,
```

Comment:
We need to define the default value since this property is optional:
```suggestion
			reverseCTAs = false,
```